v3/src/components/graph/graph-component-handler.ts

```diff
@@ -437,6 +438,7 @@ export const graphComponentHandler: DIComponentHandler = {
     updateBounds("bottom", xLowerBound, xUpperBound)
     updateBounds("left", yLowerBound, yUpperBound)
     updateBounds("rightNumeric", y2LowerBound, y2UpperBound)
+    if (rescaleAxes != null) content.rescale()
```

Comment:
I think this will rescale if `false` is passed for `rescaleAxes`, which seems wrong. Maybe you should check for truthiness instead of non-null.

I also think it's worth considering if this should happen in an `update` request. This seems like the sort of thing that might make sense as a `notify` request, which generally work as commands rather than setting values like `update`.

Reply:
>I also think it's worth considering if this should happen in an update request.

That sounds right to me now that you mention it.